Sources/ZcashLightClientKit/Tor/TorClient.swift

```diff
@@ -53,6 +53,95 @@ public class TorClient {
         return TorClient(runtimePtr: isolatedPtr)
     }
 
+    /// Changes the client's current dormant mode, putting background tasks to sleep or waking
+    /// them up as appropriate.
+    ///
+    /// This can be used to conserve CPU usage if you aren’t planning on using the client for
+    /// a while, especially on mobile platforms.
+    ///
+    /// - Parameter mode: what level of sleep to put a Tor client into.
+    public func setDormant(mode: TorDormantMode) throws {
```

Comment:
TorDormantMode is not public nor Swift enum
```swift
typedef enum TorDormantMode {
  /**
   * The client functions as normal, and background tasks run periodically.
   */
  Normal,
  /**
   * Background tasks are suspended, conserving CPU usage. Attempts to use the client will
   * wake it back up again.
   */
  Soft,
} TorDormantMode;
```

so it's not possible to use this public func atm, (it's in the tests and that is probably a reason why is this overlooked)

Reply:
Suggested by @LukasKorba in Slack:
```suggestion
    func setDormant(mode: TorDormantMode) throws {
```
(which I presume is still sufficient access to use this method inside the SDK itself).